tracer/contrib/go-redis/tracedredis.go

```diff
@@ -0,0 +1,111 @@
+package tracedredis
+
+import (
+	"context"
+	"github.com/DataDog/dd-trace-go/tracer"
+	"github.com/go-redis/redis"
+	"strconv"
+	"strings"
+)
+
+func NewTracedClient(opt *redis.Options, t *tracer.Tracer, service string) *TracedClient {
+	var host, port string
+	addr := strings.Split(opt.Addr, ":")
+	host = addr[0]
+	port = addr[1]
+	db := strconv.Itoa(opt.DB)
+
```

Comment:
also need to call to `t.SetServiceInfo(...)` here. this registers the service as a redis / db type.
`t.SetServiceInfo(service, "redis", ext.AppTypesDB)`